src/org/usfirst/frc/team4571/robot/commands/MillCommand.java

```diff
@@ -0,0 +1,48 @@
+package org.usfirst.frc.team4571.robot.commands;
+
+import org.usfirst.frc.team4571.robot.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+/**
+ *
+ */
+public class MillCommand extends Command {
+
+    public MillCommand() {
+        // Use requires() here to declare subsystem dependencies
+        // eg. requires(chassis);
+    	requires(Robot.MILL_SUBSYSTEM);
+    }
+    public void forward(){
+    	Robot.MILL_SUBSYSTEM.forward(SmartDashboard.getDouble("MillForwardSpeed", 0.5));
+    }
+    public void reverse(){
+    	Robot.MILL_SUBSYSTEM.reverse(SmartDashboard.getDouble("MillReverseSpeed", -0.5));
+    }
+    // Called just before this Command runs the first time
+    protected void initialize() {
+    }
+
+    // Called repeatedly when this Command is scheduled to run
+    protected void execute() {
```

Comment:
We should have the ability to say what method the execute() method calls, whether forward(), reverse() or stop(). In your current implementation, there is no way to call the reverse of stop method in the execute() method because it is always calling forward(). Think about how you can implement what i said